app/src/main/res/values/colors.xml

```diff
@@ -4,4 +4,5 @@
     <color name="colorPrimaryDark">#303F9F</color>
     <color name="colorAccent">#404081</color>
     <color name="colorIcon">#7F7F7F</color>
+    <color name="menuIconColor">#fff</color>
```

Comment:
The reason I made it grey if there is not enough space for the search and edit icon the icon will appear in the options menu and it is going to be invisible because it will be white on white background. I think in android should be another solution for this problem but I do not exactly know which one.

Reply:
The grey icons don't look good at all. Also there is always going to me enough space for those two icons even in smaller screen size.

Reply:
Please merge my code if possible.